plugins/generic/pln/classes/form/PLNSettingsForm.inc.php

```diff
@@ -68,6 +68,33 @@ function readInputData() {
 			$this->setData('terms_of_use_agreement', $terms_agreed);
 		}
 	}
+	
+	/**
+	 * Check for the prerequisites for the plugin, and return a translated 
+	 * message for each missing requirement.
+	 * 
+	 * @return array
+	 */
+	function _checkPrerequisites() {
+		$messages = array();
+		
+		if( ! @include_once('Archive/Tar.php')) {
```

Comment:
I think you're right @asmecher - there's no need to do the other checks if php5 isn't available.